include/library/spdm_transport_storage_lib.h

```diff
@@ -0,0 +1,281 @@
+/**
+ *  Copyright Notice:
+ *  Copyright 2025 DMTF. All rights reserved.
+ *  License: BSD 3-Clause License. For full text see link: https://github.com/DMTF/libspdm/blob/main/LICENSE.md
+ **/
+
+#ifndef STORAGE_TRANSPORT_LIB_H
+#define STORAGE_TRANSPORT_LIB_H
+
+#include "library/spdm_common_lib.h"
+#include "library/spdm_crypt_lib.h"
+
+#define LIBSPDM_STORAGE_MAX_RANDOM_NUMBER_COUNT 32
+
+/*
+ * SPDM Storage transport binding header for request encoding.
+ * This virtual header is not specific to any particular storage type, i.e
+ * SCSI, NVMe or ATA. Instead, it is used to encode requests (host to controller),
+ * to communicate SPDM storage transport information. This information shall then
+ * be used to generate the storage protocol specific commands.
+ * Refer to the storage specification (NVMe/SCSI/ATA) for storage command block
+ * descriptions.
+ *
+ * This header *shall not* be transmitted as a part of the libspdm
+ * message in the storage command block data buffer, instead, be used ONLY as a
+ * means to generate the storage specific command(s).
+ *
+ * +-----------------+--------+-------------------+
+ * |      TYPE       |Security|      Security     |
+ * |                 |Protocol| Protocol Specific |
+ * +-----------------+--------+-------------------+
+ * |Security Protocol|    1   |         2         |
+ * +-----------------+--------+-------------------+
+ */
+#pragma pack(1)
+typedef struct {
+    uint8_t security_protocol;
+    uint16_t security_protocol_specific;
+} spdm_storage_transport_virtual_header_t;
```

Comment:
we should use `libspdm_storage_transport_virtual_header_t` - the prefix is `libspdm`, which means libspdm implementation.